includes/templates/admin/edit-story.php

```diff
@@ -35,8 +35,9 @@
 
 // Preload common data.
 $preload_paths = [
-	sprintf( '/web-stories/v1/%s/%s?context=edit', $rest_base, $post->ID ),
+	sprintf( '/web-stories/v1/%s/%s?_embed=author&context=edit', $rest_base, $post->ID ),
```

Comment:
`&_locale=user` is not needed. With it, it results in an REST API request. See this screenshot.

<img width="1884" alt="Screenshot 2020-11-11 at 10 54 19" src="https://user-images.githubusercontent.com/237508/98803547-dbc10680-240c-11eb-9d8a-f7ff3ae71191.png">